npm-shrinkwrap.json

```diff
@@ -1,1109 +0,0 @@
-{
-  "name": "codeclimate-eslint",
-  "version": "0.0.3",
-  "dependencies": {
-    "acorn": {
-      "version": "3.3.0",
```

Comment:
I think this is too many lines deleted. I think it means you didn't `npm install` before `npm shrinkwrap`